hxinsight-extension/src/main/java/org/alfresco/hxi_connector/hxi_extension/rest/api/model/QuestionModel.java

```diff
@@ -52,6 +52,8 @@ public class QuestionModel
     @NotBlank
     private String question;
     @NotBlank
+    private String agentId;
+    @NotBlank
```

Comment:
Will those validation annotations work in JAMP?

Reply:
No - I raised a second PR immediately afterwards to take them out again!